homework/index.js

```diff
@@ -44,7 +72,7 @@
         return;
       }
       const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      repos.sort(sortFun).forEach(repos => renderRepoDetails(repos, ul));
```

Comment:
Since the function body of your `sortFun()` consist of a single line only, you could just use an inline arrow function like this:

```js
repos
  .sort((a, b) => a.name.localeCompare(b.name))
  .forEach(repo => renderRepoDetails(repo, repoContainer));
```